data/sql/updates/pending_db_world/rev_1555929516205604900.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1555929516205604900');
+
+-- Coilskar defender updated flag to Only Swim
+UPDATE `creature_template` SET `unit_flags`='32768' WHERE `entry`=19762;
+-- Coilskar defender remove extra flag
+UPDATE `creature_template` SET `flags_extra`='0' WHERE `entry`=19762;
```

Comment:
ideally we should:

1) use a single `UPDATE` statement 
2) just use numbers (and not strings) for numeric values

```sql
UPDATE `creature_template` SET `unit_flags`=32768, `flags_extra`=0 WHERE `entry`=19762;
```

3) ...and also, it is a good practice to properly remove/add flags using bitwise operators (but I don't know exactly which flags you are adding/removing here)

Reply:
Hmm, okay. I'll update it :D @FrancescoBorzi 
For flags, i just removed UNIT_FLAG_NOT_SELECTABLE ( Can't be selected by mouse or with /target {name} command.) - That was issue if player wanted to attack that creature ( Can't target/select him ). Dunno, but for me it looks like wrong flag

Reply:
it's fine, I will write a tutorial about how to better deal with flags. But this can be merged already.

Reply:
Okay :+1: